### ISSUE TITLE
Output probability of post processing (sigmoid instead of softmax)

### ISSUE THREAD

**Issue by aldipiroli** [open-mmlab/OpenPCDet#1250]
Hello,
In the function   [post_processing(self, batch_dict)](https://github.com/open-mmlab/OpenPCDet/blob/633db6bea843f8422382fc0ce5fd519fd264184d/pcdet/models/detectors/detector3d_template.py#L178)  used to generate the prediction dict during inference, I noticed that the output probabilities are processed using a `sigmoid` function and not a `softmax` one.
Here is the code [line](https://github.com/open-mmlab/OpenPCDet/blob/633db6bea843f8422382fc0ce5fd519fd264184d/pcdet/models/detectors/detector3d_template.py#L217):
```
if not batch_dict['cls_preds_normalized']:
    cls_preds = torch.sigmoid(cls_preds)
```
Since the output in NxC with N (num anchors) and C (num classes), applying the sigmoid to the output will result in each single logit being processed by the sigmoid function independently of the other logits of that specific anchor. 
Also, the total sum of processed logits will be different from 1, and large negative logits will simply result in 0. 

Is there a specific reason for this?

**Comment by GaneshAdam** [comment]
is there any specific reason using sigmoid over softmax?
